.travis.yml

```diff
@@ -116,56 +116,64 @@ matrix:
         - CXX1=g++-5
     - os: osx
       env:
+        - LLVM_VERSION="3.6.2"
         - LLVM_CONFIG="llvm-config-3.6"
         - config=debug
         - CC1=clang-3.6
         - CXX1=clang++-3.6
     - os: osx
       env:
+        - LLVM_VERSION="3.6.2"
         - LLVM_CONFIG="llvm-config-3.6"
         - config=release
         - lto=no
         - CC1=clang-3.6
         - CXX1=clang++-3.6
     - os: osx
       env:
+        - LLVM_VERSION="3.7.1"
         - LLVM_CONFIG="llvm-config-3.7"
         - config=debug
         - CC1=clang-3.7
         - CXX1=clang++-3.7
     - os: osx
       env:
+        - LLVM_VERSION="3.7.1"
         - LLVM_CONFIG="llvm-config-3.7"
         - config=release
         - lto=no
         - CC1=clang-3.7
         - CXX1=clang++-3.7
     - os: osx
       env:
+        - LLVM_VERSION="3.8.1"
         - LLVM_CONFIG="llvm-config-3.8"
         - config=debug
         - CC1=clang-3.8
         - CXX1=clang++-3.8
     - os: osx
       env:
+        - LLVM_VERSION="3.8.1"
         - LLVM_CONFIG="llvm-config-3.8"
         - config=release
         - lto=no
         - CC1=clang-3.8
         - CXX1=clang++-3.8
-    # LLVM 3.9 can be enabled for OSX once Homebrew supports it.
-    #- os: osx
-    #  env:
-    #    - LLVM_CONFIG="llvm-config-3.9"
-    #    - config=debug
-    #    - CC1=clang-3.9
-    #    - CXX1=clang++-3.9
-    #- os: osx
-    #  env:
-    #    - LLVM_CONFIG="llvm-config-3.9"
-    #    - config=release
-    #    - CC1=clang-3.9
-    #    - CXX1=clang++-3.9
+    - os: osx
+      env:
+        - LLVM_VERSION="3.9.0"
+        - LLVM_CONFIG="llvm-config-3.9"
```

Comment:
I think it will cause problems with the tool names in the Makefile. I'll try with a symlink.